src/app/course/components/editor/editor.component.ts

```diff
@@ -15,6 +15,8 @@ export enum CourseEditorMode {
 export class CourseEditorComponent implements OnInit {
 	@Input() public mode: CourseEditorMode;
 	@Input() public courseId?: number;
+	@Input() public doAfterSave: (Course) => Promise<void> = Promise.resolve;
+
 	public titleSubmit: string = 'Сохранить';
 	public titleEditor: string = 'Создание';
```

Comment:
Can't check this point:
![image](https://user-images.githubusercontent.com/81073/71118384-0ef11a80-21e9-11ea-9c07-724854207a04.png)
due to an exception